src/push/request.ts

```diff
@@ -57,9 +57,14 @@ export async function sendReqAndHandleError<T>(
   options: APIRequestOptions
 ): Promise<T> {
   const { statusCode, body } = await sendRequest(options);
-  return await (await handleError(statusCode, options.url, body)).json();
+  return (await (await handleError(statusCode, options.url, body)).json()) as T;
```

Comment:
```suggestion
  return (await handleError(statusCode, options.url, body)).json() as Promise<T>;
```

Could we do this to avoid the double `await`? It is not beautiful.

Reply:
I have no clue, why it was this way. thanks for spotting. I will push this manually to satifsy the prettier.